samples/old/shadow.py

```diff
@@ -0,0 +1,358 @@
+# Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Do we want to fix the generator's license template?

Reply:
Yeah let's do that